apps/ck/modules/app/app.c

```diff
@@ -354,7 +354,18 @@ Return Value:
                                Argument0,
                                Suffix);
 
-        if ((PrintLength < sizeof(Path)) && (access(Path, X_OK) == 0)) {
+        if ((PrintLength < sizeof(Path)) 			
+			
+			//
+			// execute permission - unsupported in windows
+			//
+
+#ifndef _WINDOWS_
```

Comment:
This is very ugly, I think we can do better. The problem is that X_OK is not defined, right? Maybe towards the top of the file, we can define X_OK to R_OK if it is not already defined.